x/auth/client/cli/sign.go

```diff
@@ -122,35 +125,58 @@ func makeSignCmd(cdc *amino.Codec) func(cmd *cobra.Command, args []string) error
 		if err != nil {
 			return err
 		}
+
 		defer fp.Close()
 		fmt.Fprintf(fp, "%s\n", json)
+
 		return
 	}
 }
 
-func printSignatures(stdTx auth.StdTx) bool {
+func printAndValidateSigs(cliCtx context.CLIContext, chainID string, stdTx auth.StdTx) bool {
```

Comment:
We should write a test for this function in this PR to confirm that these validations operate as expected

Reply:
It would be difficult to do this (at least when `--offline` is not provided) as it needs to reach out to a node. So I've added a new CLI test: `TestGaiaCLIValidateSignatures`.

Reply:
@rigelrozanski yes, I suppose I could do that. But to be honest, I think that's really completely overkill in this scenario...the CLI test already covers majority of the grounds.

Reply:
yeah we don't need it here right now I agree - it's already tested - but we should eventually revamp many of our tests this way